connectors/sources/sharepoint_online.py

```diff
@@ -1846,6 +1845,18 @@ async def site_list_items(
                     list_item_attachment[
                         "_original_filename"
                     ] = list_item_attachment.get("FileName", "")
+                    if (
+                        "ServerRelativePath" in list_item_attachment
+                        and "DecodedUrl" in list_item_attachment["ServerRelativePath"]
```

Comment:
```suggestion
                        and "DecodedUrl" in list_item_attachment.get("ServerRelativePath", "")
```
if it's `None` for some reason, this would error.

```
>>> "foo" in None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'NoneType' is not iterable
```

Reply:
Yeah, but I check it in the line above - `"ServerRelativePath" in list_item_attachment`, is it not sufficient?